bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/filter/core/Filter.scala

```diff
@@ -213,10 +214,10 @@ object Filter {
     def label = "RA"
     def lowest = RA.MinValue
     def highest = RA.MaxValue
-    def getter = {o: Obs => o.getRa/15.0 }
+    def getter = (o: Obs, ctx: QvContext) => o.raDeg(ctx) / 15.0
```

Comment:
This is the main change, I think.  I couldn't find the actual definition of `raDeg` but I'm guessing it does what I expect it does.